doc/nrf/releases/release-notes-1.6.0.rst

```diff
@@ -0,0 +1,471 @@
+.. _ncs_release_notes_160:
+
+Changes in |NCS| v1.6.0
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
+It includes the TF-M, MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.
+
+The |NCS| is where you begin building low-power wireless applications with Nordic Semiconductor nRF52, nRF53, and nRF91 Series devices.
+nRF53 Series devices are now supported for production.
+Wireless protocol stacks and libraries may indicate support for development or support for production for different series of devices based on verification and certification status. See the release notes and documentation for those protocols and libraries for further information.
+
+Highlights
+**********
+
+* Added production support for nRF5340 for Bluetooth LE in single protocol configuration and in multiprotocol operation with Thread for Bluetooth Peripheral role.
+* Added production support for multiple advertising sets to SoftDevice Controller.
+* Added features available in Trusted Firmware-M (TF-M) which is supported for development with nRF5340 and nRF9160.
+* Integrated Apple Find My supported for production.
+* Integrated Pelion cloud client supported for development for LTE-M/NB-IoT and Thread.
+* Edge Impulse Machine Learning has been further integrated with an Application example available for Thingy:52.
+* Added complete support for Thread 1.2 mandatory features for routers and end devices supported for development for the nRF52 family.
```

Comment:
```suggestion
* Added complete support for Thread 1.2 mandatory features for routers and end devices supported for development for the nRF52 Series family.
```

What does `supported for development` refer to here, @edmont ? "complete support" as in previous bullet points or "routers and end devices", as the context would suggest?

Reply:
Maybe like this is better explained:
> Added support for all Thread 1.2 mandatory features...